docs/governance.md

```diff
@@ -4,57 +4,57 @@ title: "Governance"
 
 # Governance
 
-Bittensor's governance protocol transitions the management of the network from centralization within the foundation to community-ownership over time. 
+Bittensor's governance protocol transitions the management of the network from centralization within the foundation to community ownership over time.
 
 The first stage of this transition to decentralized management is the creation of a bicameral legislature. In this stage, the [Triumvirate](./glossary.md#triumvirate) creates proposals for the [Senate](./senate.md) to approve. 
 
 Triumvirate members are Opentensor Foundation employees, while the Senate is formed from the top K delegate hotkeys.
 
 ## Proposals
 
-Proposals are encapsulations of other extrinsics. A proposal will be executed only after meeting the both the below two conditions:
+Proposals are encapsulations of other extrinsic and will be executed only after meeting both of the two conditions:
```

Comment:
This should be "extrinsics" (plural) because they are referring to a set of methods. A method, as in a Python method or a function, is usually referred in Bittensor as "extrinsic". So let's keep plural.